build.gradle

```diff
@@ -47,3 +43,11 @@ subprojects {
         }
     }
 }
+
+
+def getSecondDotPosition(String version) {
```

Comment:
We had a discussion about this before my leave: these type of functions are fragile and I don't want these in code unless unavoidable.

I checked and the usages are:
1. overrides.gradle: This file is _meant_ to have version written out, so here it's completely unneeded. Just write it out: `alfresco_min_version = '23.2.0'` 
1. alfresco/build.gradle: Here it is for a description. This is nice to have it readable, but you can use `alfresco_min_version` instead. 

After these, delete this function.